cmd/helm/template_test.go

```diff
@@ -37,24 +33,9 @@ func TestTemplateCmd(t *testing.T) {
 		},
 		{
 			name:   "check set name",
-			cmd:    fmt.Sprintf("template '%s' -x '%s' --set service.name=apache", chartPath, filepath.Join("templates", "service.yaml")),
+			cmd:    fmt.Sprintf("template '%s' --set service.name=apache", chartPath),
```

Comment:
Add `-x` flag back?

Reply:
According to @technosophos this was intentional in preparation for the Lua engine. He wasn't sure if we wanted to allow partial template execution as it may mess up how the Lua engine depends on certain templates. However, we currently don't have Lua engine support so this could be premature. I'm happy to add this back and we can revisit that conversation when we add Lua support in here though. What do you think about that?